Applications/MaterialManager/README.md

```diff
@@ -0,0 +1,16 @@
+# HOMAG materialManager Client
+
+The HOMAG materialManager Client enables access to the materialManager application for integration into your own applications. The functions offered on the website are available for a programmatic integration.
+The client is also available as a NuGet package on nuget.org and can be smoothly integrated into .NET applications.
+
+The HOMAG materialManager Client is written in C# ([Source Code](./../Client/IntelliDivideClient.cs)). It facilitates access to the materialManager REST API. The source code can be used to convert the relevant functions for integration into another programming language.
+
+The most important functions are described in the Documentation below, more can be found [here](./homagconnect.materialmanager.client.materialmanagerclient.md) and in the [Source Code](./../Client/MaterialManagerClient.cs) on GitHub.
+
+## Table of content
+
+1. [Authorization](Samples/Authentication/Readme.md)
+2. [Samples](Samples/README.md)
+3. [Excel (analog to MmrMobile)](../MmrMobile/Documentation/Excel/README.md)
```

Comment:
why are here links to MMR Mobile?

Reply:
Link to other Material docu is necessary I think?

Reply:
I did not add this, it was already there. As I understand it, we dont have documentation for this but because it is similar to MMR mobile, we have this link